Mathlib/Init/ZeroOne.lean

```diff
@@ -28,7 +28,7 @@ class Zero.{u} (α : Type u) where
 instance (priority := 300) Zero.toOfNat0 {α} [Zero α] : OfNat α (nat_lit 0) where
   ofNat := ‹Zero α›.1
 
-instance (priority := 200) Zero.ofOfNat0 {α} [OfNat α (nat_lit 0)] : Zero α where
+instance (priority := 10) Zero.ofOfNat0 {α} [OfNat α (nat_lit 0)] : Zero α where
   zero := 0
```

Comment:
Do we even need this instance? This looks like a performance trap with no benefits to me.